src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputLineLengthValidateImportStatements.java

```diff
@@ -0,0 +1,26 @@
+/*
+LineLength
+fileExtensions = (default)all files
+ignorePattern = ^package.*
+max = 60
+
+
+*/
+package com.puppycrawl.tools.checkstyle.checks.sizes.linelength; // ok
+
+// violation below
+import com.puppycrawl.tools.checkstyle.grammar.comments.InputFullOfSinglelineComments;
```

Comment:
Can we import some jdk class ? To keep violation comment on same line.
It will be compact.
Please add violation message to comment.

Applicable to all other cases like this.

Reply:
The reason why I wrote violation comments on different lines is due to the fact that its length is also included in the total length calculation.
eg-
```
import java.nio.channels.spi.*; // violation 'longer than 60 characters .* length is 88'
```
the length of import itself is 31 characters, the rest is contributed by the comment, which isn't a very good example IMO.

Reply:
Good point.
Let's use 'violation above ' but with message.
Please add somewhere a comment to worn users to not use trailing comment.

Reply:
Done.